contracts/modules/claims/Incidents.sol

```diff
@@ -131,40 +130,57 @@ contract Incidents is MasterAware {
     emit IncidentAdded(productId, incidentDate, priceBefore);
   }
 
+  function redeemPayoutForMember(
+    uint coverId,
+    uint incidentId,
+    uint coveredTokenAmount,
+    address member
+  ) external onlyInternal returns (uint claimId, uint payoutAmount, address payoutToken) {
+    (claimId, payoutAmount, payoutToken) = _redeemPayout(coverId, incidentId, coveredTokenAmount, member);
+  }
+
   function redeemPayout(
     uint coverId,
     uint incidentId,
     uint coveredTokenAmount
-  ) external returns (uint claimId, uint payoutAmount) {
+  ) external returns (uint claimId, uint payoutAmount, address payoutToken) {
+    (claimId, payoutAmount, payoutToken) = _redeemPayout(coverId, incidentId, coveredTokenAmount, msg.sender);
+  }
 
+  function _redeemPayout(
+    uint coverId,
+    uint incidentId,
+    uint coveredTokenAmount,
+    address owner
```

Comment:
Please rename `owner` to something like `callingMember` / `caller` / `msgSender` since this is an untrusted input and we don't know if this is the owner yet.